lib/src/__tests__/DateTimePickerTestingLib.test.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { utilsToUse } from './test-utils';
+import { TextField } from '@material-ui/core';
+import { DesktopDateTimePicker } from '../DateTimePicker';
+import { createClientRender } from './createClientRender';
+import { fireEvent, screen } from '@testing-library/react';
+
+describe('DateTimePicker testing lib', () => {
```

Comment:
What do you think of naming it

```jsx
describe('<DateTimePicker />', () => {
```

While renaming the older tests with "legacy" inside them? :)